src/bot.ts

```diff
@@ -103,64 +103,117 @@ export class DiscordBot {
         const client = await this.clientFactory.getClient();
 
         if (!this.config.bridge.disableTypingNotifications) {
-            client.on("typingStart", async (c, u) => this.OnTyping(c, u, true) );
-            client.on("typingStop", async (c, u) => this.OnTyping(c, u, false) );
+            client.on("typingStart", async (c, u) => {
+                try {
+                    await this.OnTyping(c, u, true);
+                } catch (err) { log.error("typingStart", err); }
```

Comment:
Some of these could be warnings, because someone on error logging probably doesn't care if the odd typing notif fails.